tests/route/test_route_perf.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+import json
+from datetime import datetime
+from tests.common.utilities import wait_until
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+ROUTE_TABLE_NAME = 'ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY'
+
+def prepare_dut(duthost, intf_neighs):
+    for intf_neigh in intf_neighs:
+        # Set up interface
+        duthost.shell('sudo config interface ip add {} {}'.format(intf_neigh['interface'], intf_neigh['ip']))
+        
+        # Set up neighbor
+        duthost.shell('sudo ip neigh add {} lladdr {} dev {}'.format(intf_neigh['neighbor'], intf_neigh['mac'], intf_neigh['interface']))
+
+def cleanup_dut(duthost, intf_neighs):
+    for intf_neigh in intf_neighs:
+        # Delete neighbor
+        duthost.shell('sudo ip neigh del {} dev {}'.format(intf_neigh['neighbor'], intf_neigh['interface']))
+
+        # remove interface
+        duthost.shell('sudo config interface ip remove {} {}'.format(intf_neigh['interface'], intf_neigh['ip']))
+
+def generate_intf_neigh(num_neigh):
+    # Generate interfaces and neighbors
+    intf_neighs = []
+    str_intf_nexthop = {'ifname':'', 'nexthop':''}
+    for idx_neigh in range(num_neigh):
+        intf_neigh = {
+            'interface' : 'Ethernet%d' % (idx_neigh * 4 + 4),
+            'ip' : '10.%d.0.1/24' % (idx_neigh + 1),
+            'neighbor' : '10.%d.0.2' % (idx_neigh + 1),
+            'mac' : '55:54:00:ad:48:%0.2x' % idx_neigh
```

Comment:
`55` as the first byte would cause issue in some platforms as they treat this as a multicast mac. For e,g `01:00:53` is the standard multicast mac but many platforms checks only the first byte `01`. So please change it to `54` or `00`

Reply:
Thanks for the suggestion! Fixed.